diff -rupN gnome-sharp-2.24.1_orig/art/art-sharp-2.0.pc.in gnome-sharp-2.24.1/art/art-sharp-2.0.pc.in
--- gnome-sharp-2.24.1_orig/art/art-sharp-2.0.pc.in	2010-04-10 20:46:25.000000000 +0000
+++ gnome-sharp-2.24.1/art/art-sharp-2.0.pc.in	2010-04-10 20:48:06.000000000 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
diff -rupN gnome-sharp-2.24.1_orig/gconf/GConf/gconf-sharp-2.0.pc.in gnome-sharp-2.24.1/gconf/GConf/gconf-sharp-2.0.pc.in
--- gnome-sharp-2.24.1_orig/gconf/GConf/gconf-sharp-2.0.pc.in	2010-04-10 20:46:25.000000000 +0000
+++ gnome-sharp-2.24.1/gconf/GConf/gconf-sharp-2.0.pc.in	2010-04-10 20:53:20.000000000 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 
diff -rupN gnome-sharp-2.24.1_orig/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in gnome-sharp-2.24.1/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in
--- gnome-sharp-2.24.1_orig/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in	2010-04-10 20:46:25.000000000 +0000
+++ gnome-sharp-2.24.1/gconf/GConf.PropertyEditors/gconf-sharp-peditors-2.0.pc.in	2010-04-10 20:54:47.000000000 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 
diff -rupN gnome-sharp-2.24.1_orig/gnome/gnome-sharp-2.0.pc.in gnome-sharp-2.24.1/gnome/gnome-sharp-2.0.pc.in
--- gnome-sharp-2.24.1_orig/gnome/gnome-sharp-2.0.pc.in	2010-04-10 20:46:25.000000000 +0000
+++ gnome-sharp-2.24.1/gnome/gnome-sharp-2.0.pc.in	2010-04-10 20:55:27.000000000 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
diff -rupN gnome-sharp-2.24.1_orig/gnomevfs/gnome-vfs-sharp-2.0.pc.in gnome-sharp-2.24.1/gnomevfs/gnome-vfs-sharp-2.0.pc.in
--- gnome-sharp-2.24.1_orig/gnomevfs/gnome-vfs-sharp-2.0.pc.in	2010-04-10 20:46:25.000000000 +0000
+++ gnome-sharp-2.24.1/gnomevfs/gnome-vfs-sharp-2.0.pc.in	2010-04-10 20:55:54.000000000 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gapi-2.0
