--- a/server/src/storage/dbconfigpostgresql.cpp	2010-07-31 16:12:25.000000000 +0000
+++ b/server/src/storage/dbconfigpostgresql.cpp	2010-10-03 10:23:29.421345015 +0000
@@ -67,7 +67,8 @@
     const QStringList postgresSearchPath = QStringList()
       << QLatin1String( "/usr/sbin" )
       << QLatin1String( "/usr/local/sbin" )
-      << QLatin1String( "/usr/lib/postgresql/8.4/bin" );
+      << QLatin1String( "/usr/lib/postgresql/8.4/bin" )
+      << QLatin1String( "/usr/local/bin" );
 
     defaultServerPath = XdgBaseDirs::findExecutableFile( QLatin1String( "pg_ctl" ), postgresSearchPath );
     defaultInitDbPath = XdgBaseDirs::findExecutableFile( QLatin1String( "initdb" ), postgresSearchPath );
