diff -Naur a/rabbitvcs/__init__.py b/rabbitvcs/__init__.py
--- a/rabbitvcs/__init__.py	2011-02-18 13:04:33.000000000 +0000
+++ b/rabbitvcs/__init__.py	2011-02-23 21:09:29.155866770 +0000
@@ -36,7 +36,7 @@
 TEMP_DIR_PREFIX = "rabbitvcs-"
 LOCALE_DIR = "%s/locale" % os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 if not os.path.exists(LOCALE_DIR):
-    LOCALE_DIR = "/usr/share/locale"
+    LOCALE_DIR = "/usr/local/share/locale"
 
 WEBSITE = "http://www.rabbitvcs.org/"
 
diff -Naur a/rabbitvcs/ui/about.py b/rabbitvcs/ui/about.py
--- a/rabbitvcs/ui/about.py	2010-12-10 00:06:15.000000000 +0000
+++ b/rabbitvcs/ui/about.py	2011-02-23 21:09:29.155866770 +0000
@@ -60,7 +60,7 @@
         self.about.set_website("http://www.rabbitvcs.org")
         self.about.set_website_label("http://www.rabbitvcs.org")
         
-        doc_path_root = "/usr/share/doc"
+        doc_path_root = "/usr/local/share/doc"
         doc_path_regex = re.compile("rabbitvcs")
         for dir in os.listdir(doc_path_root):
             if doc_path_regex.search(dir):
diff -Naur a/rabbitvcs/ui/create.py b/rabbitvcs/ui/create.py
--- a/rabbitvcs/ui/create.py	2010-12-19 14:00:23.000000000 +0000
+++ b/rabbitvcs/ui/create.py	2011-02-23 21:09:29.155866770 +0000
@@ -46,7 +46,7 @@
             os.makedirs(path)
         
         # Let svnadmin return a bad value if a repo already exists there
-        ret = subprocess.call(["/usr/bin/svnadmin", "create", path])
+        ret = subprocess.call(["/usr/local/bin/svnadmin", "create", path])
         if ret == 0:
             rabbitvcs.ui.dialog.MessageBox(_("Repository successfully created"))
         else:
diff -Naur a/rabbitvcs/ui/settings.py b/rabbitvcs/ui/settings.py
--- a/rabbitvcs/ui/settings.py	2010-12-10 00:06:15.000000000 +0000
+++ b/rabbitvcs/ui/settings.py	2011-02-23 21:09:29.155866770 +0000
@@ -291,7 +291,7 @@
 
     def on_external_diff_tool_browse_clicked(self, widget):
         chooser = rabbitvcs.ui.dialog.FileChooser(
-            _("Select a program"), "/usr/bin"
+            _("Select a program"), "/usr/local/bin"
         )
         path = chooser.run()
         path = path.replace("file://", "")
diff -Naur a/rabbitvcs/util/configspec/configspec.ini b/rabbitvcs/util/configspec/configspec.ini
--- a/rabbitvcs/util/configspec/configspec.ini	2010-02-26 15:43:55.000000000 +0000
+++ b/rabbitvcs/util/configspec/configspec.ini	2011-02-23 21:09:29.155866770 +0000
@@ -6,7 +6,7 @@
 show_debug = boolean(default=False)
 
 [external]
-diff_tool = string(default="/usr/bin/meld")
+diff_tool = string(default="/usr/local/bin/meld")
 diff_tool_swap = boolean(default=False)
 
 [cache]
diff -Naur a/rabbitvcs/util/settings.py b/rabbitvcs/util/settings.py
--- a/rabbitvcs/util/settings.py	2011-02-10 22:25:42.000000000 +0000
+++ b/rabbitvcs/util/settings.py	2011-02-23 21:10:47.665784985 +0000
@@ -71,7 +71,7 @@
     configspec_paths = [
         os.path.join(dirname(__file__), "configspec/configspec.ini"),
         os.path.join(package_prefix(), "share/rabbitvcs/configspec.ini"),
-        "/usr/share/rabbitvcs/configspec.ini"
+        "/usr/local/share/rabbitvcs/configspec.ini"
     ]
     
     for path in configspec_paths:
@@ -95,7 +95,7 @@
             
         Set settings:
             sm = SettingsManager()
-            sm.set("external", "diff_tool", "/usr/bin/meld")
+            sm.set("external", "diff_tool", "/usr/local/bin/meld")
             sm.write()
     """
     
