diff -Naur a/cherrytree b/cherrytree
--- a/cherrytree	2011-02-20 12:44:05.000000000 +0000
+++ b/cherrytree	2011-04-02 22:58:30.258667689 +0000
@@ -23,7 +23,7 @@
 import sys, os, gtk, gettext
 
 if os.path.isdir('modules'): MODULES_PATH = os.path.join(os.curdir, 'modules')
-else: MODULES_PATH = '/usr/share/cherrytree/modules/'
+else: MODULES_PATH = '/usr/local/share/cherrytree/modules/'
 
 sys.path.append(MODULES_PATH)
 
diff -Naur a/linux/cherrytree.desktop b/linux/cherrytree.desktop
--- a/linux/cherrytree.desktop	2011-02-20 12:44:05.000000000 +0000
+++ b/linux/cherrytree.desktop	2011-04-02 22:59:36.841993633 +0000
@@ -11,7 +11,7 @@
 Comment[ru]=Записная книжка с иерархической структурой
 Comment[uk]=Записник з ієрархічною структурою
 Exec=cherrytree %f
-Icon=/usr/share/icons/hicolor/scalable/apps/cherrytree.svg
+Icon=cherrytree
 MimeType=text/cherrytree-ctd;text/cherrytree-ctz;
 Terminal=false
 Type=Application
diff -Naur a/modules/cons.py b/modules/cons.py
--- a/modules/cons.py	2011-03-07 17:14:33.000000000 +0000
+++ b/modules/cons.py	2011-04-02 23:00:19.155323504 +0000
@@ -42,8 +42,8 @@
       GLADE_PATH = 'glade/'
       LOCALE_PATH = 'locale/'
    else:
-      GLADE_PATH = '/usr/share/cherrytree/glade/'
-      LOCALE_PATH = '/usr/share/locale/'
+      GLADE_PATH = '/usr/local/share/cherrytree/glade/'
+      LOCALE_PATH = '/usr/local/share/locale/'
 AVAILABLE_LANGS = ['default', 'cs', 'de', 'en', 'es', 'fr', 'it', 'pl', 'ru', 'uk']
 SHOW_MENU_ICONS = "gconftool-2 --set /desktop/gnome/interface/menus_have_icons --type bool 1"
 COLOR_WHITE = "#ffffffffffff"
