diff -Naur NetworkManager-0.8.0.999_orig/src/backends/NetworkManagerTinyCore.c NetworkManager-0.8.0.999/src/backends/NetworkManagerTinyCore.c
--- NetworkManager-0.8.0.999_orig/src/backends/NetworkManagerTinyCore.c	1970-01-01 00:00:00.000000000 +0000
+++ NetworkManager-0.8.0.999/src/backends/NetworkManagerTinyCore.c	2010-07-04 18:26:44.946228258 +0000
@@ -0,0 +1,60 @@
+/* NetworkManager -- Network link manager
+ *
+ * Implementation for the Tiny Core Linux distro - http://www.tinycorelinux.com
+ *
+ * Sercan Arslan <arslanserc@gmail.com>
+ *
+ * Based on NetworkManagerSlackware.c by Narayan Newton <narayan_newton@yahoo.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * (C) Copyright 2010 Sercan Arslan
+ */
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "NetworkManagerGeneric.h"
+#include "nm-system.h"
+
+/*
+ * nm_system_enable_loopback
+ *
+ * Bring up the loopback interface
+ *
+ */
+void nm_system_enable_loopback (void)
+{
+	nm_generic_enable_loopback ();
+}
+
+
+/*
+ * nm_system_update_dns
+ *
+ * Make glibc/nscd aware of any changes to the resolv.conf file by
+ * restarting nscd.
+ *
+ */
+void nm_system_update_dns (void)
+{
+	/* I'm not running nscd */
+}
+
