--- a/data/etc/sysactions.conf.in	2012-11-03 21:20:53.000000000 +0000
+++ b/data/etc/sysactions.conf.in	2012-11-16 13:51:52.033310113 +0000
@@ -43,13 +43,13 @@
 # user and group name can use glob matches (* == all for example) like the
 # shell. as can action names allowed or denied. 
 
-action:   halt      /sbin/shutdown -h now
-action:   reboot    /sbin/shutdown -r now
+action:   halt      [ $BACKUP = 1 ] && filetool.sh -b; exitcheck.sh
+action:   reboot    [ $BACKUP = 1 ] && filetool.sh -b; exitcheck.sh reboot
 action:   suspend   @SUSPEND@
 action:   hibernate @HIBERNATE@
-action:   /bin/mount /bin/mount
-action:   /bin/umount /bin/umount
-action:   /usr/bin/eject /usr/bin/eject
+action:   /usr/local/bin/mount /usr/local/bin/bin/mount
+action:   /usr/local/bin/umount /usr/local/bin/umount
+action:   /usr/local/usr/bin/eject /usr/local/usr/bin/eject
 action:   gdb       gdb
 
 # on FreeBSD use this instead of the above.
