--- a/src/main.c
+++ b/src/main.c
@@ -176,9 +176,9 @@
 
   /* run with a controlled path */
   if (helper_dir != NULL)
-      path = g_strdup_printf ("%s:" PACKAGE_LIBEXEC_DIR ":/sbin:/bin:/usr/sbin:/usr/bin", helper_dir);
+      path = g_strdup_printf ("%s:" PACKAGE_LIBEXEC_DIR ":/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin", helper_dir);
   else
-      path = g_strdup (PACKAGE_LIBEXEC_DIR ":/sbin:/bin:/usr/sbin:/usr/bin");
+      path = g_strdup (PACKAGE_LIBEXEC_DIR ":/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin");
 
   if (!g_setenv ("PATH", path, TRUE))
     {
