diff -Naur a/desktop/base.desktop.in.in b/desktop/base.desktop.in.in
--- a/desktop/base.desktop.in.in	2011-01-06 17:15:46.000000000 +0000
+++ b/desktop/base.desktop.in.in	2011-02-02 20:10:23.898959226 +0000
@@ -2,7 +2,7 @@
 Version=1.0
 Terminal=false
 TryExec=lobase@OOO_BINSUFFIX@
-Exec=oobase@OOO_BINSUFFIX@ %U
+Exec=lobase@OOO_BINSUFFIX@ %U
 Icon=ooo-base@OOO_BINSUFFIX@
 Type=Application
 Categories=Office;Database;
diff -Naur a/desktop/calc.desktop.in.in b/desktop/calc.desktop.in.in
--- a/desktop/calc.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/calc.desktop.in.in	2011-02-02 20:10:30.845625473 +0000
@@ -2,7 +2,7 @@
 Version=1.0
 Terminal=false
 TryExec=localc@OOO_BINSUFFIX@
-Exec=oocalc@OOO_BINSUFFIX@ %U
+Exec=localc@OOO_BINSUFFIX@ %U
 Icon=ooo-calc@OOO_BINSUFFIX@
 Type=Application
 InitialPreference=6
diff -Naur a/desktop/draw.desktop.in.in b/desktop/draw.desktop.in.in
--- a/desktop/draw.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/draw.desktop.in.in	2011-02-02 20:10:41.053119904 +0000
@@ -2,7 +2,7 @@
 Version=1.0
 Terminal=false
 TryExec=lodraw@OOO_BINSUFFIX@
-Exec=oodraw@OOO_BINSUFFIX@ %U
+Exec=lodraw@OOO_BINSUFFIX@ %U
 Icon=ooo-draw@OOO_BINSUFFIX@
 Type=Application
 Categories=Graphics;VectorGraphics;
diff -Naur a/desktop/impress.desktop.in.in b/desktop/impress.desktop.in.in
--- a/desktop/impress.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/impress.desktop.in.in	2011-02-02 20:10:45.565623868 +0000
@@ -2,7 +2,7 @@
 Version=1.0
 Terminal=false
 TryExec=loimpress@OOO_BINSUFFIX@
-Exec=ooimpress@OOO_BINSUFFIX@ %U
+Exec=loimpress@OOO_BINSUFFIX@ %U
 Icon=ooo-impress@OOO_BINSUFFIX@
 Type=Application
 Categories=Office;Presentation;
diff -Naur a/desktop/math.desktop.in.in b/desktop/math.desktop.in.in
--- a/desktop/math.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/math.desktop.in.in	2011-02-02 20:10:50.789819885 +0000
@@ -3,7 +3,7 @@
 Terminal=false
 NoDisplay=true
 TryExec=lomath@OOO_BINSUFFIX@
-Exec=oomath@OOO_BINSUFFIX@ %U
+Exec=lomath@OOO_BINSUFFIX@ %U
 Icon=ooo-math@OOO_BINSUFFIX@
 Type=Application
 Categories=Office;Math;
diff -Naur a/desktop/template.desktop.in.in b/desktop/template.desktop.in.in
--- a/desktop/template.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/template.desktop.in.in	2011-02-02 20:11:05.069788526 +0000
@@ -3,7 +3,7 @@
 Terminal=false
 NoDisplay=true
 TryExec=lofromtemplate@OOO_BINSUFFIX@
-Exec=oofromtemplate@OOO_BINSUFFIX@
+Exec=lofromtemplate@OOO_BINSUFFIX@
 Icon=ooo-gulls@OOO_BINSUFFIX@
 Type=Application
 Categories=X-SuSE-Core-Office;Office;
diff -Naur a/desktop/web.desktop.in.in b/desktop/web.desktop.in.in
--- a/desktop/web.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/web.desktop.in.in	2011-02-02 20:11:12.098975221 +0000
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-TryExec=ooweb@OOO_BINSUFFIX@
+TryExec=loweb@OOO_BINSUFFIX@
 Exec=loweb@OOO_BINSUFFIX@ %U
 Icon=ooo-web@OOO_BINSUFFIX@
 Type=Application
diff -Naur a/desktop/writer.desktop.in.in b/desktop/writer.desktop.in.in
--- a/desktop/writer.desktop.in.in	2011-01-06 17:15:47.000000000 +0000
+++ b/desktop/writer.desktop.in.in	2011-02-02 20:11:17.595620446 +0000
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Version=1.0
 Terminal=false
-TryExec=oowriter@OOO_BINSUFFIX@
+TryExec=lowriter@OOO_BINSUFFIX@
 Exec=lowriter@OOO_BINSUFFIX@ %U
 Icon=ooo-writer@OOO_BINSUFFIX@
 Type=Application
