--- a/download.in	2011-01-10 14:21:52.000000000 +0000
+++ c/download.in	2011-01-27 18:03:28.374570577 +0000
@@ -286,13 +286,14 @@
 		}
 
         system( "cd @CLONEDIR@/$what ; git status | grep '^# Your branch is'" );
-        system( "for file in `cd git-hooks ; echo *`
+        system( "cd @CLONEDIR@/$what
+                 for file in `cd .git/hooks ; echo *`
                  do
                      cd @CLONEDIR@/$what
                      hook=\".git/hooks/\$file\"
                      if [ ! -x \"\$hook\" -a ! -L \"\$hook\" ] ; then
                          rm -f \"\$hook\"
-                         ln -s \"../../../../git-hooks/\$file\" \"\$hook\"
+                         ln -s \"../../../../.git/hooks/\$file\" \"\$hook\"
                      fi
                  done" );
     }
