diff -Naur a/configure.in b/configure.in
--- a/configure.in	2011-01-18 19:59:16.000000000 +0000
+++ b/configure.in	2011-01-27 15:33:20.048802002 +0000
@@ -765,6 +765,7 @@
         OOo4Kids*) VENDORNAME="EducOOo" ;;
         OpenOffice.org*) VENDORNAME="OpenOffice.org" ;;
         OxygenOffice*) VENDORNAME="OxygenOffice" ;;
+        TinyCoreLinux*) VENDORNAME="Tiny Core Linux" ;;
         *) AC_MSG_WARN([Cannot guess vendor from distro - please add it to configure.in]) ;;
 esac
 AC_MSG_RESULT([$VENDORNAME])
@@ -1792,6 +1793,7 @@
 distro-configs/LibreOfficeLinuxDevel.conf
 distro-configs/LibreOfficeMacOSX.conf
 distro-configs/LibreOfficeOpenBSD.conf
+distro-configs/TinyCoreLinux.conf
 etc/Makefile
 extensions/Makefile
 extensions/tweakconfig/Makefile
diff -Naur a/distro-configs/Makefile.in b/distro-configs/Makefile.in
--- a/distro-configs/Makefile.in	2011-01-18 19:59:19.000000000 +0000
+++ b/distro-configs/Makefile.in	2011-01-27 15:35:51.355870190 +0000
@@ -58,6 +58,7 @@
 	$(srcdir)/SUSE-11.2.conf.in $(srcdir)/SUSE-9.1.conf.in \
 	$(srcdir)/SUSE-9.2.conf.in $(srcdir)/SUSE-9.3.conf.in \
 	$(srcdir)/SUSE-reduced.conf.in $(srcdir)/SUSE.conf.in \
+	$(srcdir)/TinyCoreLinux.conf.in \
 	$(srcdir)/Slackware.conf.in $(srcdir)/Solaris.conf.in \
 	$(srcdir)/UbuntuHardy.conf.in $(srcdir)/UbuntuJaunty.conf.in \
 	$(srcdir)/UbuntuKarmic.conf.in $(srcdir)/UbuntuL10n.conf.in \
@@ -85,6 +86,7 @@
 	SUSE-11.0.conf SUSE-11.1.conf SUSE-11.2.conf Slackware.conf \
 	Mandriva.conf Mandriva64.conf NovellWin32.conf \
 	NovellWin32Eval.conf NovellWin32ISO.conf Gentoo.conf \
+	TinyCoreLinux.conf \
 	Frugalware.conf openmamba.conf translate-org-za-linux.conf \
 	translate-org-za-win32.conf rpl2.conf rpl2-64.conf \
 	Pardus2007.conf Pardus2008.conf PlainLinux.conf Win32.conf \
@@ -416,6 +418,8 @@
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 Slackware.conf: $(top_builddir)/config.status $(srcdir)/Slackware.conf.in
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
+TinyCoreLinux.conf: $(top_builddir)/config.status $(srcdir)/TinyCoreLinux.conf.in
+	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 Mandriva.conf: $(top_builddir)/config.status $(srcdir)/Mandriva.conf.in
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 Mandriva64.conf: $(top_builddir)/config.status $(srcdir)/Mandriva64.conf.in
diff -Naur a/distro-configs/TinyCoreLinux.conf.in b/distro-configs/TinyCoreLinux.conf.in
--- a/distro-configs/TinyCoreLinux.conf.in	1970-01-01 00:00:00.000000000 +0000
+++ b/distro-configs/TinyCoreLinux.conf.in	2011-01-27 15:38:19.589591216 +0000
@@ -0,0 +1,31 @@
+--with-vendor=\"Tiny Core Linux\"
+--disable-fontooo
+--disable-binfilter
+--disable-gnome-vfs
+--enable-odk
+--disable-qadevooo
+--disable-pam
+--enable-cups
+--enable-pdfimport
+--enable-presenter-console
+--enable-minimizer
+--enable-graphite
+--with-system-expat
+--with-system-curl
+--with-system-xrender-headers
+--with-system-freetype
+--with-system-icu
+--with-system-libxml
+--with-system-libxslt
+--with-system-mozilla=xulrunner
+--with-system-cairo
+--with-system-graphite
+--with-system-hunspell
+--with-system-db
+--with-system-boost
+--without-stlport
+--without-myspell-dicts
+--with-system-dicts
+--with-external-dict-dir=/usr/local/share/myspell
+--with-external-hyph-dir=/usr/local/share/myspell
+--with-external-thes-dir=/usr/local/share/myspell
diff -Naur a/download.in b/download.in
--- a/download.in	2011-01-10 14:21:52.000000000 +0000
+++ b/download.in	2011-01-27 15:39:33.625434239 +0000
@@ -620,6 +620,7 @@
     $distro eq 'openmamba' ||
     $distro eq 'SUSE-9.1' || $distro eq 'SUSE-9.2' || $distro eq 'SUSE-9.3' ||
     $distro eq 'Slackware' ||
+    $distro eq 'TinyCoreLinux' ||
     $distro eq 'NovellWin32' || $distro eq 'GoOoWin32' ||
     $distro eq 'OxygenOfficeLinux' || $distro eq 'OxygenOfficeWindows' ||
     $distro eq 'translate-org-za-linux' || $distro eq 'translate-org-za-win32' || $distro eq 'Pardus') {
diff -Naur a/patches/dev300/apply b/patches/dev300/apply
--- a/patches/dev300/apply	2011-01-18 14:45:11.000000000 +0000
+++ b/patches/dev300/apply	2011-01-27 15:40:13.252093853 +0000
@@ -65,6 +65,8 @@
 SUSE      : NovellBase, DisableGCJ, BerkeleyDB43, ImpressFontSize, PostgreSQL, SUSE112
 # Slackware
 Slackware : LinuxCommon, SlackwareOnly, NotDebian, Mono, msaccess
+# Tiny Core Linux
+TinyCoreLinux : LinuxCommon, NotDebian, CairoFonts, Shrink
 # Debian
 DebianBase : LinuxCommon, DebianBaseOnly, GCCSTL, BerkeleyDB43, Split, PostgreSQL
 Debian : DebianBase, DebianOnly, DebianSidOnly, Debian32Only, HPPA, AXP, kFreeBSD
