diff -Naur a/configure.in b/configure.in
--- a/configure.in	2010-12-18 19:49:35.000000000 +0000
+++ b/configure.in	2011-01-01 11:36:19.976896450 +0000
@@ -758,6 +758,7 @@
         OOo4Kids*) VENDORNAME="EducOOo" ;;
         OpenOffice.org*) VENDORNAME="OpenOffice.org" ;;
         OxygenOffice*) VENDORNAME="OxygenOffice" ;;
+        TinyCoreLinux*) VENDORNAME="Tiny Core Linux" ;;
         *) AC_MSG_WARN([Cannot guess vendor from distro - please add it to configure.in]) ;;
 esac
 AC_MSG_RESULT([$VENDORNAME])
@@ -1774,6 +1775,7 @@
 distro-configs/LibreOfficeLinuxDevel.conf
 distro-configs/LibreOfficeMacOSX.conf
 distro-configs/LibreOfficeOpenBSD.conf
+distro-configs/TinyCoreLinux.conf
 etc/Makefile
 extensions/Makefile
 extensions/tweakconfig/Makefile
diff -Naur a/distro-configs/Makefile.in b/distro-configs/Makefile.in
--- a/distro-configs/Makefile.in	2010-12-18 19:49:38.000000000 +0000
+++ b/distro-configs/Makefile.in	2011-01-01 11:44:41.897256342 +0000
@@ -58,6 +58,7 @@
 	$(srcdir)/SUSE-11.2.conf.in $(srcdir)/SUSE-9.1.conf.in \
 	$(srcdir)/SUSE-9.2.conf.in $(srcdir)/SUSE-9.3.conf.in \
 	$(srcdir)/SUSE-reduced.conf.in $(srcdir)/SUSE.conf.in \
+	$(srcdir)/TinyCoreLinux.conf.in \
 	$(srcdir)/Slackware.conf.in $(srcdir)/Solaris.conf.in \
 	$(srcdir)/UbuntuHardy.conf.in $(srcdir)/UbuntuJaunty.conf.in \
 	$(srcdir)/UbuntuKarmic.conf.in $(srcdir)/UbuntuL10n.conf.in \
@@ -83,6 +84,7 @@
 	SUSE-10.2.conf SUSE-10.3.conf SUSE-11.0.conf SUSE-11.1.conf \
 	SUSE-11.2.conf Slackware.conf Mandriva.conf Mandriva64.conf \
 	NovellWin32.conf NovellWin32Eval.conf NovellWin32ISO.conf \
+	TinyCoreLinux.conf \
 	Gentoo.conf Frugalware.conf openmamba.conf \
 	translate-org-za-linux.conf translate-org-za-win32.conf \
 	rpl2.conf rpl2-64.conf Pardus2007.conf Pardus2008.conf \
@@ -461,6 +463,8 @@
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 LibreOfficeOpenBSD.conf: $(top_builddir)/config.status $(srcdir)/LibreOfficeOpenBSD.conf.in
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
+TinyCoreLinux.conf: $(top_builddir)/config.status $(srcdir)/TinyCoreLinux.conf.in
+	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 tags: TAGS
 TAGS:
 
diff -Naur a/distro-configs/TinyCoreLinux.conf.in b/distro-configs/TinyCoreLinux.conf.in
--- a/distro-configs/TinyCoreLinux.conf.in	1970-01-01 00:00:00.000000000 +0000
+++ b/distro-configs/TinyCoreLinux.conf.in	2011-01-01 11:38:41.270215256 +0000
@@ -0,0 +1,31 @@
+--with-vendor=\"Tiny Core Linux\"
+--disable-fontooo
+--disable-binfilter
+--disable-gnome-vfs
+--disable-odk
+--disable-qadevooo
+--disable-pam
+--enable-cups
+--enable-pdfimport
+--enable-presenter-console
+--enable-minimizer
+--enable-graphite
+--with-system-expat
+--with-system-curl
+--with-system-xrender-headers
+--with-system-freetype
+--with-system-icu
+--with-system-libxml
+--with-system-libxslt
+--with-system-mozilla=xulrunner
+--with-system-cairo
+--with-system-graphite
+--with-system-hunspell
+--with-system-db
+--without-stlport
+--without-myspell-dicts
+--with-system-dicts
+--with-external-dict-dir=/usr/local/share/myspell
+--with-external-hyph-dir=/usr/local/share/myspell
+--with-external-thes-dir=/usr/local/share/myspell
+--with-java
\ No newline at end of file
diff -Naur a/download.in b/download.in
--- a/download.in	2010-11-25 19:45:58.000000000 +0000
+++ b/download.in	2011-01-01 11:35:18.360645200 +0000
@@ -625,6 +625,7 @@
     $distro eq 'openmamba' ||
     $distro eq 'SUSE-9.1' || $distro eq 'SUSE-9.2' || $distro eq 'SUSE-9.3' ||
     $distro eq 'Slackware' ||
+    $distro eq 'TinyCoreLinux' ||
     $distro eq 'NovellWin32' || $distro eq 'GoOoWin32' ||
     $distro eq 'OxygenOfficeLinux' || $distro eq 'OxygenOfficeWindows' ||
     $distro eq 'translate-org-za-linux' || $distro eq 'translate-org-za-win32' || $distro eq 'Pardus') {
diff -Naur a/patches/dev300/apply b/patches/dev300/apply
--- a/patches/dev300/apply	2010-12-16 13:52:28.000000000 +0000
+++ b/patches/dev300/apply	2011-01-01 11:40:52.133529104 +0000
@@ -65,6 +65,8 @@
 SUSE      : NovellBase, DisableGCJ, BerkeleyDB43, ImpressFontSize, PostgreSQL, SUSE112
 # Slackware
 Slackware : LinuxCommon, SlackwareOnly, NotDebian, Mono, msaccess
+# Tiny Core Linux
+TinyCoreLinux : LinuxCommon, NotDebian, CairoFonts, Shrink
 # Debian
 DebianBase : LinuxCommon, DebianBaseOnly, GCCSTL, BerkeleyDB43, Split, PostgreSQL
 Debian : DebianBase, DebianOnly, DebianSidOnly, Debian32Only, HPPA, AXP, kFreeBSD
