diff -Naur a/gnomedesktop/gnome-desktop-sharp-2.0.pc.in b/gnomedesktop/gnome-desktop-sharp-2.0.pc.in
--- a/gnomedesktop/gnome-desktop-sharp-2.0.pc.in	2008-03-07 02:22:22.000000000 +0000
+++ b/gnomedesktop/gnome-desktop-sharp-2.0.pc.in	2010-12-30 23:26:52.873883457 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gnomedesktop-sharp/2.20
diff -Naur a/gnomepanel/gnome-panel-sharp-2.24.pc.in b/gnomepanel/gnome-panel-sharp-2.24.pc.in
--- a/gnomepanel/gnome-panel-sharp-2.24.pc.in	2008-08-04 14:17:12.000000000 +0000
+++ b/gnomepanel/gnome-panel-sharp-2.24.pc.in	2010-12-30 23:27:57.913898564 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gnome-panel-sharp/2.24
diff -Naur a/gnomeprint/gnome-print-sharp-2.18.pc.in b/gnomeprint/gnome-print-sharp-2.18.pc.in
--- a/gnomeprint/gnome-print-sharp-2.18.pc.in	2008-08-04 14:19:12.000000000 +0000
+++ b/gnomeprint/gnome-print-sharp-2.18.pc.in	2010-12-30 23:28:12.643875590 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gnome-print-sharp/2.18
diff -Naur a/gtkhtml/gtkhtml-sharp-3.14.pc.in b/gtkhtml/gtkhtml-sharp-3.14.pc.in
--- a/gtkhtml/gtkhtml-sharp-3.14.pc.in	2008-03-07 02:22:23.000000000 +0000
+++ b/gtkhtml/gtkhtml-sharp-3.14.pc.in	2010-12-30 23:28:23.680540928 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/gtkhtml-sharp/3.14
diff -Naur a/nautilusburn/nautilusburn-sharp.pc.in b/nautilusburn/nautilusburn-sharp.pc.in
--- a/nautilusburn/nautilusburn-sharp.pc.in	2008-03-07 02:22:21.000000000 +0000
+++ b/nautilusburn/nautilusburn-sharp.pc.in	2010-12-30 23:28:45.358039129 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/nautilusburn-sharp/2.20
diff -Naur a/rsvg/rsvg2-sharp-2.0.pc.in b/rsvg/rsvg2-sharp-2.0.pc.in
--- a/rsvg/rsvg2-sharp-2.0.pc.in	2008-03-07 02:22:21.000000000 +0000
+++ b/rsvg/rsvg2-sharp-2.0.pc.in	2010-12-30 23:28:55.903871047 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/rsvg2-sharp/2.0
diff -Naur a/vte/vte-sharp-0.16.pc.in b/vte/vte-sharp-0.16.pc.in
--- a/vte/vte-sharp-0.16.pc.in	2008-03-07 02:22:21.000000000 +0000
+++ b/vte/vte-sharp-0.16.pc.in	2010-12-30 23:29:19.984702094 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/vte-sharp/0.16
diff -Naur a/wnck/wnck-sharp-1.0.pc.in b/wnck/wnck-sharp-1.0.pc.in
--- a/wnck/wnck-sharp-1.0.pc.in	2008-03-07 02:22:22.000000000 +0000
+++ b/wnck/wnck-sharp-1.0.pc.in	2010-12-30 23:30:16.180528915 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 gapidir=${prefix}/share/wnck-sharp/2.20
