--- NetworkManager-0.8.1~git100705/src/nm-policy-hosts.c	2010-07-05 08:01:29.000000000 +0000
+++ NetworkManager-0.8.1~git100705_tinycore/src/nm-policy-hosts.c	2010-07-05 14:43:42.111073250 +0000
@@ -307,8 +307,8 @@
 	g_return_val_if_fail (hostname != NULL, FALSE);
 	g_return_val_if_fail (out_changed != NULL, FALSE);
 
-	if (!g_file_get_contents (SYSCONFDIR "/hosts", &contents, &contents_len, &error)) {
-		nm_log_warn (LOGD_DNS, "couldn't read " SYSCONFDIR "/hosts: (%d) %s",
+	if (!g_file_get_contents ("/etc/hosts", &contents, &contents_len, &error)) {
+		nm_log_warn (LOGD_DNS, "couldn't read " "/etc/hosts: (%d) %s",
 		             error ? error->code : 0,
 		             (error && error->message) ? error->message : "(unknown)");
 		g_clear_error (&error);
@@ -333,8 +333,8 @@
 
 		g_clear_error (&error);
 		/* And actually update /etc/hosts */
-		if (!g_file_set_contents (SYSCONFDIR "/hosts", new_contents->str, -1, &error)) {
-			nm_log_warn (LOGD_DNS, "couldn't update " SYSCONFDIR "/hosts: (%d) %s",
+		if (!g_file_set_contents ("/etc/hosts", new_contents->str, -1, &error)) {
+			nm_log_warn (LOGD_DNS, "couldn't update " "/etc/hosts: (%d) %s",
 			             error ? error->code : 0,
 			             (error && error->message) ? error->message : "(unknown)");
 			g_clear_error (&error);
@@ -348,7 +348,7 @@
 		/* No change required */
 		success = TRUE;
 	} else {
-		nm_log_warn (LOGD_DNS, "couldn't read " SYSCONFDIR "/hosts: (%d) %s",
+		nm_log_warn (LOGD_DNS, "couldn't read " "/etc/hosts: (%d) %s",
 		             error->code, error->message ? error->message : "(unknown)");
 		g_clear_error (&error);
 	}
