diff -Naur gamin-0.1.9/libgamin/gam_data.c gamin-0.1.9.new/libgamin/gam_data.c
--- gamin-0.1.9/libgamin/gam_data.c	2007-07-04 15:36:48.000000000 +0200
+++ gamin-0.1.9.new/libgamin/gam_data.c	2007-10-07 13:41:58.000000000 +0200
@@ -470,7 +470,7 @@
     }
     if (is_threaded > 0) {
 	pthread_mutexattr_init(&attr);
-#if defined(linux) || defined(PTHREAD_MUTEX_RECURSIVE_NP)
+#if defined(__GLIBC__) && !defined(__GNU__)
 	pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_RECURSIVE_NP);
 #else
 	pthread_mutexattr_settype(&attr, PTHREAD_MUTEX_RECURSIVE);
