Index: mjpegtools-1.9.0/configure
===================================================================
--- mjpegtools-1.9.0.orig/configure	2009-04-25 03:27:11.000000000 +0200
+++ mjpegtools-1.9.0/configure	2009-04-25 03:27:40.000000000 +0200
@@ -25998,9 +25998,9 @@
 echo $ECHO_N "checking sub-architecture settings... $ECHO_C" >&6; }
     chmod +x $srcdir/cpuinfo.sh
     if test x$have_x86cpu = xtrue; then
-       ARCHFLAGS=`$srcdir/cpuinfo.sh x86`
+       ARCHFLAGS=``
     elif test x$have_ppccpu = xtrue; then
-       ARCHFLAGS=`$srcdir/cpuinfo.sh ppc`
+       ARCHFLAGS=``
     fi
   fi
   { echo "$as_me:$LINENO: result: $ARCHFLAGS" >&5
Index: mjpegtools-1.9.0/configure.ac
===================================================================
--- mjpegtools-1.9.0.orig/configure.ac	2009-04-25 03:27:15.000000000 +0200
+++ mjpegtools-1.9.0/configure.ac	2009-04-25 03:27:29.000000000 +0200
@@ -441,9 +441,9 @@
     AC_MSG_CHECKING(sub-architecture settings)
     chmod +x $srcdir/cpuinfo.sh
     if test x$have_x86cpu = xtrue; then
-       ARCHFLAGS=`$srcdir/cpuinfo.sh x86`
+       ARCHFLAGS=``
     elif test x$have_ppccpu = xtrue; then
-       ARCHFLAGS=`$srcdir/cpuinfo.sh ppc`
+       ARCHFLAGS=``
     fi
   fi
   AC_MSG_RESULT($ARCHFLAGS)
