Index: gir-repository-0.6.5/gir/AvahiCore-0.6.gir
===================================================================
--- gir-repository-0.6.5.orig/gir/AvahiCore-0.6.gir	2010-08-11 14:28:30.589205002 +1000
+++ gir-repository-0.6.5/gir/AvahiCore-0.6.gir	2010-08-11 14:28:48.779205002 +1000
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
             xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
Index: gir-repository-0.6.5/gir/DBusGLib-1.0.gir
===================================================================
--- gir-repository-0.6.5.orig/gir/DBusGLib-1.0.gir	2010-08-11 14:28:30.599205002 +1000
+++ gir-repository-0.6.5/gir/DBusGLib-1.0.gir	2010-08-11 14:28:45.289205001 +1000
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
 	    xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
Index: gir-repository-0.6.5/gir/JSCore-1.0.gir
===================================================================
--- gir-repository-0.6.5.orig/gir/JSCore-1.0.gir	2010-08-11 14:28:30.609205002 +1000
+++ gir-repository-0.6.5/gir/JSCore-1.0.gir	2010-08-11 14:28:41.209205002 +1000
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0">
   <namespace name="JSCore" version="1.0" shared-library="webkit-1.0">
Index: gir-repository-0.6.5/gir/LibTelepathy.gir
===================================================================
--- gir-repository-0.6.5.orig/gir/LibTelepathy.gir	2010-08-11 14:28:30.629205002 +1000
+++ gir-repository-0.6.5/gir/LibTelepathy.gir	2010-08-11 14:28:39.169205000 +1000
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
 	    xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
Index: gir-repository-0.6.5/gir/TelepathyGLib.gir
===================================================================
--- gir-repository-0.6.5.orig/gir/TelepathyGLib.gir	2010-08-11 14:28:30.639205002 +1000
+++ gir-repository-0.6.5/gir/TelepathyGLib.gir	2010-08-11 14:28:35.959205001 +1000
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<repository version="1.0"
+<repository version="1.1"
             xmlns="http://www.gtk.org/introspection/core/1.0"
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
 	    xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
