--- a/Makefile.in	2013-12-05 16:10:00.666685902 +0000
+++ b/Makefile.in	2013-12-05 16:10:42.830029771 +0000
@@ -1249,7 +1249,7 @@
 @CLIENT_TRUE@			client/main.c
 
 @CLIENT_TRUE@client_connmanctl_LDADD = gdbus/libgdbus-internal.la @DBUS_LIBS@ @GLIB_LIBS@ \
-@CLIENT_TRUE@				-lreadline -ldl
+@CLIENT_TRUE@				-lreadline -lncurses -ldl
 
 unit_test_pbkdf2_sha1_SOURCES = unit/test-pbkdf2-sha1.c \
 				src/shared/sha1.h src/shared/sha1.c
