--- a/plugins/growisofs/burn-dvd-rw-format.c	2010-06-21 22:56:08.000000000 +0000
+++ b/plugins/growisofs/burn-dvd-rw-format.c	2010-07-29 12:17:57.157270280 +0000
@@ -222,6 +222,6 @@
 	brasero_plugin_test_app (plugin,
 	                         "dvd+rw-format",
 	                         "-v",
-	                         "* BD/DVD±RW/-RAM format utility by <appro@fy.chalmers.se>, version %d.%d",
+	                         "* %*s format utility by <appro@fy.chalmers.se>, version %d.%d.",
 	                         version);
 }
