--- a/src/gdu/Makefile.am	2010-01-18 17:24:03.000000000 +0000
+++ b/src/gdu/Makefile.am	2010-07-28 12:01:10.766089219 +0000
@@ -14,20 +14,20 @@
 gdu-marshal.c: gdu-marshal.list
 	echo "#include \"gdu-marshal.h\"" > $@ && glib-genmarshal $< --prefix=gdu_marshal --body >> $@
 
-udisks-daemon-glue.h: /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.xml Makefile.am
-	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-daemon-glue.h /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.xml
+udisks-daemon-glue.h: /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.xml Makefile.am
+	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-daemon-glue.h /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.xml
 
-udisks-device-glue.h: /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Device.xml Makefile.am
-	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-device-glue.h /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Device.xml
+udisks-device-glue.h: /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Device.xml Makefile.am
+	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-device-glue.h /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Device.xml
 
-udisks-adapter-glue.h: /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Adapter.xml Makefile.am
-	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-adapter-glue.h /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Adapter.xml
+udisks-adapter-glue.h: /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Adapter.xml Makefile.am
+	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-adapter-glue.h /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Adapter.xml
 
-udisks-expander-glue.h: /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Expander.xml Makefile.am
-	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-expander-glue.h /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Expander.xml
+udisks-expander-glue.h: /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Expander.xml Makefile.am
+	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-expander-glue.h /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Expander.xml
 
-udisks-port-glue.h: /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Port.xml Makefile.am
-	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-port-glue.h /usr/share/dbus-1/interfaces/org.freedesktop.UDisks.Port.xml
+udisks-port-glue.h: /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Port.xml Makefile.am
+	dbus-binding-tool --prefix=devkit_disks_daemon --mode=glib-client --output=udisks-port-glue.h /usr/local/share/dbus-1/interfaces/org.freedesktop.UDisks.Port.xml
 
 lib_LTLIBRARIES=libgdu.la
 
