--- a/contrib/run-conkeror	2010-07-22 23:26:57.891947935 +0000
+++ b/contrib/run-conkeror	2010-07-22 23:41:20.538500947 +0000
@@ -15,24 +15,15 @@
 #
 
 ## locate conkeror
-thisscript=$(readlink -f "$0")
-contrib=${thisscript%/*}
-conkeror=${contrib%/*}
+
+conkeror=/usr/local/conkeror
+xulrunner=/usr/local/bin/xulrunner
 
 ## locate xulrunner
-versions=${CONKEROR_XULRUNNER_VERSION:-$(grep -h '^\[' "$HOME"/.gre.d/*.conf /etc/gre.d/*.conf 2>/dev/null \
-    | sed -e 's/^\[\(.*\)\]/\1/' \
-    | sort -urV)}
-for version in ${versions}; do
-    xrdir=$(grep -Fxh -A2 -m1 "[$version]" "$HOME"/.gre.d/*.conf /etc/gre.d/*.conf 2>/dev/null |\
-            grep GRE_PATH |\
-            cut -d= -f2)
-    xulrunner="${xrdir%/}/xulrunner"
-    if [ -x "$xulrunner" ]; then
-        exec "$xulrunner" "$conkeror"/application.ini "$@"
-    fi
-done
 
-## if we get this far, we didn't find xulrunner
+if [ -x "$xulrunner" ]; then
+     exec "$xulrunner" "$conkeror"/application.ini "$@"
+fi
+
 echo "$0: cannot continue. XULRunner not found." >&2
 exit 1
