diff -Naur a/data/midori.desktop.in b/data/midori.desktop.in
--- a/data/midori.desktop.in	2011-11-10 23:21:06.000000000 +0000
+++ b/data/midori.desktop.in	2011-11-11 11:09:00.546604914 +0000
@@ -12,20 +12,3 @@
 StartupNotify=true
 X-Osso-Type=application/x-executable
 X-Osso-Service=midori
-X-Ayatana-Desktop-Shortcuts=TabNew;WindowNew;Private
-
-[TabNew Shortcut Group]
-Name=New _Tab
-Exec=midori -e TabNew
-TargetEnvironment=Unity
-
-[WindowNew Shortcut Group]
-Name=New _Window
-Exec=midori -e WindowNew
-TargetEnvironment=Unity
-
-[Private Shortcut Group]
-Name=New P_rivate Browsing Window
-Exec=midori --private
-TargetEnvironment=Unity
-
