diff -Naur a/data/mint.pc.in b/data/mint.pc.in
--- a/data/mint.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mint.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,5 +1,5 @@
-prefix=${pcfiledir}/../..
-exec_prefix=${pcfiledir}/../..
+prefix=@prefix@
+exec_prefix=${prefix}
 libdir=${prefix}/lib
 includedir=${prefix}/include/mono-@API_VER@
 
diff -Naur a/data/mono-2.pc.in b/data/mono-2.pc.in
--- a/data/mono-2.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono-2.pc.in	2010-12-27 21:23:24.620846345 +0000
@@ -1,5 +1,5 @@
-prefix=${pcfiledir}/../..
-exec_prefix=${pcfiledir}/../..
+prefix=@prefix@
+exec_prefix=${prefix}
 libdir=${prefix}/@reloc_libdir@
 includedir=${prefix}/include/mono-@API_VER@
 sysconfdir=@sysconfdir@
diff -Naur a/data/mono-cairo.pc.in b/data/mono-cairo.pc.in
--- a/data/mono-cairo.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono-cairo.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,5 +1,5 @@
-prefix=${pcfiledir}/../..
-exec_prefix=${pcfiledir}/../..
+prefix=@prefix@
+exec_prefix=${prefix}
 libdir=${prefix}/@reloc_libdir@
 includedir=${prefix}/include
 
diff -Naur a/data/mono-lineeditor.pc.in b/data/mono-lineeditor.pc.in
--- a/data/mono-lineeditor.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono-lineeditor.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 assemblies_dir=${prefix}/lib/mono-source-libs
 Sources=${assemblies_dir}/getline.cs
 
diff -Naur a/data/mono-nunit.pc.in b/data/mono-nunit.pc.in
--- a/data/mono-nunit.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono-nunit.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib
 
diff -Naur a/data/mono-options.pc.in b/data/mono-options.pc.in
--- a/data/mono-options.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono-options.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,4 +1,4 @@
-prefix=${pcfiledir}/../..
+prefix=@prefix@
 assemblies_dir=${prefix}/lib/mono-source-libs
 Sources=${assemblies_dir}/Options.cs
 
diff -Naur a/data/mono.pc.in b/data/mono.pc.in
--- a/data/mono.pc.in	2010-11-12 10:24:41.000000000 +0000
+++ b/data/mono.pc.in	2010-12-27 21:22:58.947515527 +0000
@@ -1,5 +1,5 @@
-prefix=${pcfiledir}/../..
-exec_prefix=${pcfiledir}/../..
+prefix=@prefix@
+exec_prefix=${prefix}
 libdir=${prefix}/@reloc_libdir@
 includedir=${prefix}/include/mono-@API_VER@
 sysconfdir=@sysconfdir@
