diff -Naur a/src/VBox/Additions/x11/vboxmouse/Makefile.kmk b/src/VBox/Additions/x11/vboxmouse/Makefile.kmk
--- a/src/VBox/Additions/x11/vboxmouse/Makefile.kmk	2011-10-03 14:32:35.000000000 +0000
+++ b/src/VBox/Additions/x11/vboxmouse/Makefile.kmk	2011-10-04 17:55:11.000000000 +0000
@@ -237,6 +237,21 @@
 
 endif # neq ($(KBUILD_TARGET),linux)
 
+#
+# vboxmouse_drv_111
+#
+
+DLLS += vboxmouse_drv_111
+vboxmouse_drv_111_TEMPLATE = VBOXGUESTR3XORGMOD
+vboxmouse_drv_111_DEFS := $(vboxmouse_drv_70_DEFS) NO_ANSIC
+vboxmouse_drv_111_INCS := \
+	/usr/local/include/x11 \
+	/usr/local/include/xorg \
+	/usr/local/include/pixman-1 \
+	$(PATH_SUB_CURRENT)
+vboxmouse_drv_111_SOURCES = \
+	vboxmouse_15.c
+
 # Check the undefined symbols in the X.Org modules against lists of allowed
 # symbols.  Not very elegant, but it will catch problems early.
 ifdef VBOX_WITH_TESTCASES
